action.yml

```diff
@@ -27,7 +27,7 @@ inputs:
     required: false
     description: |-
       If a branch name does not match this regex return `defaultTag` or empty instead.
-    default: "^.+$"
+    default: "^(?![.-])[a-zA-Z0-9_.-]{1,128}$"
```

Comment:
https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md?plain=1#L158-L160
suggests
`[a-zA-Z0-9_][a-zA-Z0-9._-]{0,127}`
which I think is clearer

Reply:
```suggestion
    default: "^[a-zA-Z0-9_][a-zA-Z0-9._-]{0,127}$"
```

Reply:
Ah, nice reference - happy there are clear open specifications available